docs/guides/platform/logs.mdx

```diff
@@ -32,12 +32,12 @@ Nango provides detailed filtering and searching of log messages by integration,
 
 We recommend exploring the logs in your own Nango account under **Logs**.
 
-## OpenTelemetry logs export
+## OpenTelemetry export
 
-Nango supports exporting logs with OpenTelemetry.
+Nango supports exporting logs as OpenTelemetry traces.
```

Comment:
Should we just say `Nango supports exporting OpenTelemetry traces` and remove the mention of logs?